src/redux/appState.js

```diff
@@ -0,0 +1,29 @@
+// -- Constants --------------------------------------- //
+const APP_STATE_UPDATE = 'contacts/APP_STATE_UPDATE';
+// -- Actions ---------------------------------------- //
+
+export const appUpdate = state => (dispatch, getState) => {
+  const { app } = getState();
+  const updatedState = {
```

Comment:
This seems a bit confusing/ redundant given the action below, and it breaks the pattern with the way we normally do our updates. Can we have it such that APP_STATE_UPDATE action below just takes in the walletReady value as the action.payload, then you don't need to do the getState or updatedState here, but just pass along walletReady as the payload! (and you can either destructure state as ({walletRead}) or just accept walletReady as the boolean value as is